crates/uv/tests/it/pip_sync.rs

```diff
@@ -1702,7 +1702,7 @@ fn install_path_source_dist_cached() -> Result<()> {
     ----- stdout -----
 
     ----- stderr -----
-    Removed 19 files for source-distribution ([SIZE])
```

Comment:
Why did we remove these?

Reply:
After adding the package cleaning progress bar, to show summary for each package, we have to collect each summary into a HashMap and display them at the end. For simplicity (or perhaps laziness), I've decided to show only the total summary instead. :)